js-exercises/is-triangle/isTriangle.js

```diff
@@ -0,0 +1,53 @@
+function isNumber(...args) {
+  let result = true;
+
+  for (const arg of args) {
+    if (typeof arg !== 'number') {
+      result = false;
+      break;
+    }
+  }
+
+  return result;
+}
+
+function isPositiveInteger(...args) {
+  let result = true;
+
+  for (const arg of args) {
+    if (arg !== Math.abs(arg)) {
+      result = false;
+      break;
```

Comment:
`break` is not a good idea at all.

Reply:
As we found alternate, so this is not longer used now